src/lib/telemetry/TelemetryReporter.ts

```diff
@@ -117,16 +139,39 @@ export class TelemetryReporter {
       }
       return error
     })
+    await ensureDir(join(TelemetryReporter.PENDING_DATA_DIR, 'errors'))
     await ensureFile(metaErrorFilePath)
```

Comment:
Just out of curiosity, this `ensureFile` is not necessary, `writeJson` will create in case it doesn't exists. I think i will not create the directory though, so `ensureDir` is necessary.